docs/convertRaMessagesToI18next.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
I wouldn't add a specific doc chapter about it - the package README should be enough